server/cmwell-data-tools/src/main/scala/cmwell/tools/data/utils/akka/HttpConnections.scala

```diff
@@ -19,60 +19,29 @@ import akka.http.scaladsl._
 import akka.http.scaladsl.settings.{ClientConnectionSettings, ConnectionPoolSettings}
 import akka.stream.Materializer
 import cmwell.tools.data.utils.logging.LabelId
-import com.typesafe.config.ConfigFactory
 
 object HttpConnections extends DataToolsConfig {
 
   def outgoingConnection(host: String, port: Int, protocol: String = "http")(implicit system: ActorSystem,
-                                                                             label: Option[LabelId] = None) = {
-    val userAgent = label.fold(s"cmwell-data-tools using akka-http/${config.getString("akka.version")}")(
```

Comment:
It's my mistake I told you to remove it. It has the label that is injected to the header.
It's up to you whether to leave the label-in-header as a debugging aid or not.